tasks/main.yml

```diff
@@ -18,6 +18,7 @@
     group: root
     mode: 0644
   notify: restart nfs
+  when: nfs_disable_server == false
```

Comment:
This condition also needs to be added to the next task, in an `or` condition (after the check for a non-empty list of mounts).